Tests/SonarScanner.MSBuild.PreProcessor.Tests/Infrastructure/MockSonarQubeServer.cs

```diff
@@ -33,9 +33,12 @@ internal class MockSonarQubeServer : ISonarQubeServer
     {
         private readonly IList<string> calledMethods;
 
+        private readonly IList<string> warnings;
+
         public MockSonarQubeServer()
         {
             this.calledMethods = new List<string>();
+            this.warnings = new List<string>();
```

Comment:
_Out of curiosity:_ Why not use initializer instead? All 3 items could be initialized to simplify the structure and the constructor could be removed.
```
private readonly IList<string> warnings = new List<string>();
```

Reply:
You can do the same with the property to remove the explicit constructor.
```
public ServerDataModel Data { get; set; } = new ServerDataModel();
```